sites/fast-website/src/app/components/fast-frame/fast-frame.ts

```diff
@@ -170,8 +173,8 @@ export class FastFrame extends FASTElement {
         this.backgroundColor = this.previewBackgroundPalette[this.lastSelectedIndex];
     };
 
-    private resetExpanded = (): void => {
-        this.expanded = false;
+    private resetExpandedResponsive = (): void => {
+        (this.expanded = false), (this.isResponsive = !this.isResponsive);
```

Comment:
If the drawer breakpoint is hit, then `isResponsive` is inverted? Can you explain this logic? Seems like you get the reverse depending on if you are starting with a width of greater than the drawer breakpoint vs starting with less than the drawer breakpoint.

Reply:
I might be calling the observable, `isResponsive` by a wrong name, and that might make this confusing. But the main goal I am trying to achieve here is to make sure `tabindex` on the controls is always 0 for when its in full-width, beyond the mobile breakpoint. Then when it hits the mobile breakpoint, the tabindex can go to -1. This is what `isResponsive` is doing. Then at the mobile breakpoint, `expanded` will take over the tabindex.
Currently, we only have the observable, `expanded`, toggling the `tabindex` to 0 and -1, but only when it is at the mobile breakpoint.

Reply:
I think the sticking point is "when the mobile breakpoint is hit" which this could either start off as being in the mobile breakpoint, or it could start larger than the mobile breakpoint and the user could resize the window and hit it. These seem like the produce inverted values because all this does is reverse the boolean. Shouldn't this just be setting it to `false` or whatever the case may need to be? Why reverse whatever it was initially.

Reply:
Didn't think about it starting off on mobile. Did some restructuring, created a property, called `mql`, set that as the windows.matchMedia breakpoint, and added the event listener to that.
In addition, renamed `isResponsive` to `isMobile` and created a `setTabIndex` property.
Now when you start off in mobile or full-width, the tabindex is set to match the window size.
Thanks to John for helping me out with this.